narratives/ncov_sit-rep_2020-05-15.md

```diff
@@ -0,0 +1,227 @@
+
+These mutations can be used as "bread crumbs" to trace the history of a given sample. Just as the people in your family share unique combinations of genetic characteristics, closely related viral samples share unique combinations of genetic mutations.
+<br><br>
+For example, here we show the viral "family tree" with each sample colored by which amino acid is present at a specific location in the viral genome (gene "ORF1a", site 57). 
+<br><br>
+Highlighting samples from New York, we see that most of the cases from New York and Europe have the amino acid Histidine ("H") rather than Glutamine ("Q") at this position. This, in combination with all the other sites in the genome, tells us that these cases are all closely related to each other.
+<br><br>
+You can also see the position in gene "ORF3a" highlighted on the chart at the bottom of the left.
+<br><br>
+For a more in-depth explanation of these concepts, we recommend [this visual explanation](https://www.nytimes.com/interactive/2020/04/30/science/coronavirus-mutations.html) by Jonathan Corum and Carl Zimmer.
+
+
+<!-- ############ SLIDE BREAK ############# -->
+<!-- This is left-side text -->
+# [The mutation rate of SARS-CoV-2 is typical for a coronavirus](https://nextstrain.org/ncov/global/2020-05-14?c=num_date&d=tree,entropy&l=clock&p=full)
+
+Because mutations are biologically inevitable, they tend to accumulate at a steady rate over time.
+<br><br>
+Here, the x axis shows the date that each sample was collected. The y axis shows the *total number* of mutations each strain is away from the root of the tree. Each sample is colored by the date it was collected.
+<br><br>
+While there are certainly some outliers, on average, we see changes accumulate at a rate of ~24 substitutions per year. This means that if a single viral lineage were transmitting from one person to the next for a full year, we'd expect its entire genome to accumulate roughly 24 substitutions by the end of the year. 
+As the whole genome is ~30,000 bases, this corresponds to only (24/30,000)\*100\% = ~0.08\% of the viral genome changing each year.
+<br><br>
+For context, SARS (also a coronavirus) accumulates XX substitutions per year (changing XX\% of its genome), while influenza accumulates XX substitutions per year (changing XX\% of its genome).
+
+
+<!-- ############ SLIDE BREAK ############# -->
+<!-- This is left-side text -->
+# [As far as we know, there is only 1 strain of SARS-CoV-2](https://nextstrain.org/ncov/global/2020-05-14?branchLabel=clade&c=clade_membership&d=tree&p=full)
+
+There have been many reports about multiple "strains" of SARS-CoV-2.
+First, a clarification: when virologists use the word "strain," it's often just as a useful label so we can all refer to the same group of samples (e.g., our clade labels, shown here). This does *not* imply any known functional difference.
```

Comment:
```suggestion
What is referred to as a "strain", can differ between people using the term and doesn't always mean that there is a functional difference between viruses from different "strains".
We here consider different strains to be different viruses that exhibit different behavior. 
This could, for example, be the case for viruses having different clinical outcomes, different speed of transmission or the ability to re-infect people that are immune to different strains.
```